css-media-queries/meta-viewport.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <!-- your code here 👇 -->
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CSS Media Queries - Meta Viewport</title>
+    <link rel="stylesheet" href="reset.css" />
+    <style>
+      .container {
+        min-height: 100vh;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+
+      .box {
+        width: 200px;
+        height: 200px;
+        background-color: blue;
+      }
+
+      @media (min-width: 568px) {
+        .box {
+          background-color: red;
+        }
+      }
+    </style>
```

Comment:
We should be specifying that we are targetting the screen in the media query. We want to be more specific so that it is not targeting for other available media types (braille, handheld, projection, speech. . .)